### ISSUE TITLE
Would it be possible to create a Quick Access or keyboard shortcuts to quickly apply ICC profiles rather than having to use NVIDIA Controller presets?

### ISSUE THREAD

**Issue by Jason-GitH** [Maassoft/ColorControl#329]
I have a random issues with the NVIDIA Controller presets causing random cool color temp in Windows. I think this probably has to do with HDMI handshake on my LG C2. I was wondering if there could be a way to incorporate shortcuts or Quick Access menu to just quickly change ICC profiles and bypass the need for NVIDIA Controller presets.

**Comment by Maassoft** [comment]
Yes, I'm already thinking about adding a "System Controller" where you can configure presets to automate various system settings (like color profiles). But it might take a while before that is completed.
In the mean time, as long as you leave everything "Excluded" (leave "Included" check mark unchecked) a NVIDIA preset should not alter anything else except the setting that is Included.
